src/sentry/static/sentry/app/components/footer.tsx

```diff
@@ -37,6 +37,12 @@ const Footer = () => {
               fixed: 'Acceptance Test',
               value: config.version.current,
             })}
+            {' ('}
+            {getDynamicText({
+              fixed: 'master',
```

Comment:
I guess this can be `test` instead of `master`

Reply:
Can we do this ^